proposals/lip-0052.md

```diff
@@ -1843,6 +1847,10 @@ def transferCrossChain(
         emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, INVALID_RECEIVING_CHAIN)
         raise Exception("Receiving chain cannot be the sending chain.")
 
+    if not Interoperability.isChannelActive(receivingChainID):
+        emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, RECEIVING_CHAIN_NOT_ACTIVE)
+        raise Exception("Receiving chain is not active.")    
```

Comment:
`  emitFailedTransferCrossChainEvent` function doesn't take `amount` as input parameter. Similar check should be done for other verify conditions.

Reply:
You're correct, I copy/pasted it from LIP0051. Thanks for noticing it!
Please check this [commit](https://github.com/LiskHQ/lips/pull/396/commits/c6365e48b27ec428cd81d1cf54f8719d8a8055aa).